ecchronos-binary/src/pylib/ecchronoslib/types.py

```diff
@@ -74,10 +75,64 @@ def get_next_repair(self):
             return "-"
         return datetime.datetime.fromtimestamp(self.next_repair_in_ms / 1000).strftime('%Y-%m-%d %H:%M:%S')
 
+class Job(object):
```

Comment:
I thought we were going for 2 different types, repairs and schedules and not having any "job"